test/component_fetcher_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule MozartFetcher.ComponentFetcherTest do
+  use ExUnit.Case
+
+  alias MozartFetcher.{ComponentFetcher}
+
+  describe "fetch" do
+    test "successful response" do
+      defmodule MockClient do
+        def get(_) do
```

Comment:
Instead of `_`, we could match on the exact string we expect to be sent to the `get` function